translations/messages.fr.xlf

```diff
@@ -3299,6 +3307,10 @@
         <source>You have selected an empty space, you must select one of your own pieces.</source>
         <target>Vous avez sélectionné une case vide, vous devez sélectionner l&apos;une de vos pièces.</target>
       </trans-unit>
+      <trans-unit id="7782275541868926318" datatype="html">
+        <source>Move cannot be static, choose different starting and ending coord.</source>
```

Comment:
je sais pas si c'est une erreur qui est affichée, mais elle mérite à être reformulée correctement en anglais :D (coord - > coordinate, move -> A move, choose -> you must choose)